node/src/components/network.rs

```diff
@@ -447,7 +447,7 @@ async fn server_task<REv: ReactorEventT<P>, P: PayloadT>(
                 // https://github.com/libp2p/rust-libp2p/issues/1876
                 swarm_event = swarm.next_event() => {
                     trace!("{}: {:?}", our_id(&swarm), swarm_event);
-                    handle_swarm_event(&mut swarm, event_queue, swarm_event, &known_addresses_mut, is_bootsrap_node).await;
+                    handle_swarm_event(&mut swarm, event_queue, swarm_event, &known_addresses_mut, is_bootstrap_node).await;
```

Comment:
Line too long. (Rustfmt has given up again?)

Reply:
Probably due to being inside the `select!` macro?  I'll fix manually.

Reply:
Fixed in [7556dd4](https://github.com/CasperLabs/casper-node/pull/1349/commits/7556dd4d9b18e489a7bbf59f2cfc302b3b852bf1).